services/core-web/src/components/common/DocumentTable.tsx

```diff
@@ -41,20 +41,18 @@ interface DocumentTableProps {
   enableBulkActions?: boolean;
   documentParent?: string;
   view?: "standard" | "minimal";
-  openModal?: (arg) => void;
+  openModal: (arg) => void;
   openDocument: any;
-  closeModal?: () => void;
+  closeModal: () => void;
   removeDocument: (event, doc_guid: string, mine_guid: string) => void;
-  archiveMineDocuments?: (mineGuid: string, mineDocumentGuids: string[]) => void;
+  archiveMineDocuments: (mineGuid: string, mineDocumentGuids: string[]) => void;
   onArchivedDocuments?: (docs?: MineDocument[]) => void;
-  onReplaceDocument?: (document: MineDocument) => void;
-  documentColumns: ColumnType<unknown>[];
+  documentColumns?: ColumnType<unknown>[];
```

Comment:
there were some props here that were completely unused that I removed (onReplaceDocument, handleRowSelectionChange) otherwise I made the reduxy ones required and some of the other ones not required.